scripts/travis/build-all-in-one-image.sh

```diff
@@ -11,7 +11,7 @@ make build-all-in-one-linux
 export REPO=jaegertracing/all-in-one
 
 docker build -f cmd/standalone/Dockerfile -t $REPO:latest .
-export CID=$(docker run -d -p 16686:16686 $REPO:latest)
+export CID=$(docker run -d -p 16686:16686 -p 5778:5778 $REPO:latest)
```

Comment:
seriously... how did this ever work then?

Reply:
The test uses the query service's instrumentation to create debug traces so it never needed agent's udp ports.